test/cryptography/sign.js

```diff
@@ -116,6 +122,25 @@ ${defaultSecondSignature}
 			});
 	});
 
+	describe('#writeVarIt', () => {
+		it('should calculate correct value for n < 253', () => {
+			return expect(writeVarInt(5)).to.be.eql(Buffer.from([5]));
+		});
+		it('should calculate correct value for n < 65536', () => {
+			return expect(writeVarInt(300)).to.be.eql(Buffer.from([253, 44, 1]));
+		});
+		it('should calculate correct value for n > 65536', () => {
+			return expect(writeVarInt(65537)).to.be.eql(Buffer.from([254, 1, 0, 1, 0]));
+		});
+	});
+
+	describe('#messageDigest', () => {
+		it('should create message digest', () => {
+			const msgBytes = messageDigest(defaultMessage);
+			return expect(msgBytes).to.be.eql(defaultMessageDigest);
+		});
+	});
```

Comment:
Ideally we would have test cases for each length range.

Reply:
Which length range?

Reply:
0–252, 253–65536, ...